# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xorg-server/glibc-2.18.patch
# Copyright (C) 2018 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- xorg-server-1.20.1/config/udev.c.vanilla	2018-08-15 20:21:39.790624585 +0000
+++ xorg-server-1.20.1/config/udev.c	2018-08-15 20:22:39.890623681 +0000
@@ -30,6 +30,7 @@
 #include <libudev.h>
 #include <ctype.h>
 #include <unistd.h>
+#include <sys/sysmacros.h>
 
 #include "input.h"
 #include "inputstr.h"
